src/components/Results.tsx

```diff
@@ -179,22 +65,24 @@ export const Results: React.FC<{
     >
       <Box flex={true} overflow="auto">
         <Box>
-          <MessageBox loading type="info" show={loading}>
+          <MessageBox loading type="info" show={true}>
             One moment...
           </MessageBox>
           <MessageBox
             loading
             type="info"
-            show={!loading && !error && filteredFacilities.length === 0}
+            show={!isFetching && !error && accommodations?.length === 0}
           >
             Could not find place
           </MessageBox>
           <MessageBox type="error" show={!!error}>
-            {error}
+            {/* Make it typesafe:https://tanstack.com/query/v4/docs/typescript */}
+            {(error as Error) && 'Something went wrong '}
```

Comment:
Not following you I think.

 `{(error as Error) || 'Something went wrong '}` you mean? 
 
 FYI: I am trying to display an error message once we have an error. I keep the error message on purpose general, because you don't want to show the error messages that you are getting from the BE to the user, this only makes them confused.

Reply:
I think more appropriate will be:

```
{(error as Error) && (error as Error).message ? (error as Error).message : 'Something went wrong '}
```

Reply:
I have implemented the suggestion of @valera-rozuvan